cumulus_library/actions/exporter.py

```diff
@@ -56,13 +70,28 @@ def export_study(
         description=f"Exporting {manifest_parser.get_study_prefix()} data...",
     ):
         query = f"SELECT * FROM {table}"
-        dataframe = db.execute_as_pandas(query)
+        # Note: we assume that, for duckdb, you are unlikely to be dealing with large
+        # exports, so it will ignore the chunksize parameter, as it does not provide
+        # a pandas enabled cursor.
+        dataframe_chunks, db_schema = db.execute_as_pandas(query, chunksize=chunksize)
+        first_chunk = next(dataframe_chunks)
```

Comment:
What's this line about? Trying to raise an iteration error? (though that would break out of this method, since it's not caught)

Reply:
you know - this is holdover from when i was trying to do this with dataframes. it can probably be cut.